docker/Dockerfile.alma-9

```diff
@@ -74,7 +74,8 @@ RUN debuginfo-install -y python3 \
 # Add development packages
 COPY requirements.txt requirements.txt
 RUN python3 -m pip install --upgrade pip \
-    && python3 -m pip install --requirement requirements.txt
+    && python3 -m venv /venv \
```

Comment:
Will this env be functional without `--system-site-packages`? I think we need that to access the deps that we install as RPMs.

Reply:
We do not need to do `--system-site-packages` in the local test venv. This shouldn't be different, no?

I tested locally with:
```
podman run \
    --env-host --privileged --rm -it \
    --volume `pwd`:/vdsm:Z \
    localhost/vdsm-test:centos-8 \
        bash -c "cd /vdsm && source /venv/bin/activate && ./ci/tests.sh"
```
And the results were mostly good. A few tests failed, but mostly due to environment issues (as the loop devices, or network stuff, or systemctl tests), that would fail also with current containers in local environment:
```
...
common/systemctl_test.py FFFF
...
lib/rpc/http_image_request_handler_test.py ..............127.0.0.1 - - [20/Dec/2022 08:31:58] "my-image-data" 400 
...
network/integration/netinfo_test.py ...s......E
network/integration/netlink_test.py ............                                                                                                                                    
network/integration/tc_test.py ........FFx
...

ERROR:   tests: commands failed
ERROR:   lib: commands failed
ERROR:   network: commands failed
  virt: commands succeeded
  gluster: commands succeeded
  hooks: commands succeeded
```
But no complaint about missing packages.

Same run with `quay.io/ovirt/vdsm-test-centos-8:latest` (and without venv activation):
```
....
ERROR:   tests: commands failed
ERROR:   lib: commands failed
ERROR:   network: commands failed
  virt: commands succeeded
  gluster: commands succeeded
  hooks: commands succeeded
```
Same results.

Reply:
I think it is probably `tox` finding the system packages.

Reply:
Right, this is possible.